src/scripts/components/VideoForm.js

```diff
@@ -234,67 +185,38 @@ class VideoForm extends Component<Props, Object> {
 
   render () {
     const video: VideoRecord = this.props.selectedVideo
-    // console.log(video.getIn(['transcodingStatus', 'data', 'result']))
-    // console.log(video.getIn(['transcodingStatus', 'data', 'result', 'screenshots']))
+
     if (!this.state.id) {
       return (
         <Card title="No video selected!">{this.props.selectedVideo.id}</Card>
       )
     }
 
     const title = video.title || video.filename
-    const duration = (video && video.get('duration')) || ''
-    let durationBox = null
-    if (duration) {
-      durationBox = (
-        <VideoMediaTime>
-          <VideoMediaTimeText>{duration}</VideoMediaTimeText>
-        </VideoMediaTime>
-      )
-    }
 
     const fileSize = prettyBytes((video && video.get('filesize')) || 0)
-    const ipfsHash = (video && video.get('ipfsHash')) || ''
-    const urlToPlay = `/play/${video.id}`
-    const urlForSharing = `https://portal.paratii.video/play/${video.id}`
-
-    const thumbImages = video && video.getIn(['thumbnails'])
-
-    let thumbImage = 'https://paratii.video/public/images/paratii-src.png'
-    if (thumbImages && ipfsHash) {
-      const firstThumb = thumbImages[0]
-      if (firstThumb !== undefined) {
-        thumbImage = `https://gateway.paratii.video/ipfs/${ipfsHash}/${firstThumb}`
-      }
-    }
-
-    const uploadProgress = video.uploadStatus.data.progress
-    const transcodingStatus = video.transcodingStatus.data.progress
-    const progress = Math.floor((uploadProgress + transcodingStatus) / 2)
-
     const isPublished = video.published === true || video.published === 'true'
     const isPublishable =
       video.transcodingStatus.name === 'success' && isPublished === false
 
-    const publishButton = ''
-    // temporarily removed the publish button until the modal is working
-    // if (isPublished) {
-    //   publishButton = ''
-    // } else {
-    //   publishButton = (
-    //     <ButtonWrapper>
-    //       <Button
-    //         id="video-submit"
-    //         type="submit"
-    //         onClick={this.onPublishSubmit}
-    //         disabled={!isPublishable}
-    //         purple
-    //       >
-    //         Publish
-    //       </Button>
-    //     </ButtonWrapper>
-    //   )
-    // }
+    let publishButton
+    if (isPublished) {
+      publishButton = ''
+    } else {
+      publishButton = (
+        <ButtonWrapper>
+          <Button
+            id="video-submit"
+            type="submit"
+            onClick={this.onPublishVideo}
+            disabled={!isPublishable}
+            purple
+          >
+            Publish
+          </Button>
+        </ButtonWrapper>
+      )
+    }
```

Comment:
☝️ the way this is written totally works, but might want to look at [this conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) pattern for the future as it is slightly more conventional.  [This one](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) is very common as well!

Reply:
this code isn't changed from this PR and commits. 
Is this the code that you want to show?:

![image](https://user-images.githubusercontent.com/320157/36982643-1916ed64-206f-11e8-8ef6-850dec1ce9dd.png)

Reply:
It was just commented out before, I see now. Fine to leave the code as is, but FYI that there are more conventional patterns for conditional rendering